atlasdb-client/src/main/java/com/palantir/atlasdb/keyvalue/impl/TracingKeyValueService.java

```diff
@@ -531,6 +531,11 @@ public ListenableFuture<Map<Cell, Value>> getAsync(TableReference tableRef, Map<
         });
     }
 
+    @Override
+    public boolean sweepsEntriesInStrictlyNonDecreasingFashion() {
+        return delegate().sweepsEntriesInStrictlyNonDecreasingFashion();
+    }
```

Comment:
Might be worth going through the KeyValueService class hierarchy - I think there are a few more that might need it (`ProfilingKeyValueService`, `TableRemappingKeyValueService` probably doesn't matter for now but could lead to weird behaviour when we try and turn this on for dbkvs)